pkg/webhooks/namespace/namespace.go

```diff
@@ -123,39 +168,40 @@ func (s *NamespaceWebhook) Authorized(request admissionctl.Request) admissionctl
 // Is the request authorized?
 func (s *NamespaceWebhook) authorized(request admissionctl.Request) admissionctl.Response {
 	var ret admissionctl.Response
+
+	// Picking OldObject or Object will suffice for most validation concerns
 	ns, err := s.renderNamespace(request)
 	if err != nil {
 		log.Error(err, "Couldn't render a Namespace from the incoming request")
 		return admissionctl.Errored(http.StatusBadRequest, err)
 	}
-	// L49-L56
-	// service accounts making requests will include their name in the group
 	for _, group := range request.UserInfo.Groups {
 		if privilegedServiceAccountsRe.Match([]byte(group)) {
 			ret = admissionctl.Allowed("Privileged service accounts may access")
 			ret.UID = request.AdmissionRequest.UID
 			return ret
 		}
 	}
-	// L58-L62
-	// This must be prior to privileged namespace check
```

Comment:
this is the second comment I think we could probably leave in

Reply:
Good call.